core/client-internal/src/main/java/tachyon/client/block/BlockStoreContext.java

```diff
@@ -270,6 +272,7 @@ public boolean hasLocalWorker() {
    * Re-initializes the {@link BlockStoreContext}. This method should only be used in
    * {@link ClientContext}.
    */
+  @SuppressFBWarnings
```

Comment:
I think we should put `synchronized` here to guard `reset`, otherwise there could me write-write race on `mLocalBlockWorkerClientPoolInitialized` in `reset`. And that's probably why you see the warning from findbugs